src/code_review_agent/config.py

```
@@ -0,0 +1,172 @@
+"""Configuration management for the code review agent."""
+
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)